### ISSUE TITLE
False positives for no_implicit_braces with @-prefixed class name

### ISSUE THREAD

**Issue by provegard** [clutchski/coffeelint#605]
I have the following code:

```
class @LinkedList
  constructor: ->
    @head = @tail = null
```

I lint with the following config:

```
{
  "no_implicit_braces": {
    "level": "warn"
  }
}
```

And I get:

    #2: Implicit braces are forbidden.

I.e. on the line with `constructor: ->`

But there are no implicit braces in the code.
